### PR TITLE
Added org.guitarix.Guitarix

### PR DESCRIPTION
**Pull request by hfiguiere**
This is Guitarix.

It require JACK over pipewire.

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build org.guitarix.Guitarix

**Comment by hfiguiere** [comment]
bot, build org.guitarix.Guitarix

**Comment by hfiguiere** [comment]
bot, build org.guitarix.Guitarix

File: org.guitarix.Guitarix.json [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
+        /* Note playback */
+        "--socket=pulseaudio",
+        /* MIDI */
+        "--device=all",
+        /* For Jack over pipewire */
+        "--socket=system-bus",
```
Comment (GeorgesStavracas) [inline org.guitarix.Guitarix.json:19]: Is this needed? On OBS Studio, there's no system-bus permission set and JACK still works

Comment (hfiguiere) [reply org.guitarix.Guitarix.json:19]: As I remember it was needed but it is not impossible that this changed. I'll test again without.

**Comment by hfiguiere** [comment]
bot, build org.guitarix.Guitarix

File: org.guitarix.Guitarix.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```
Comment (barthalion) [inline org.guitarix.Guitarix.appdata.xml:2]: Doesn't it need `<launchable>`? Also, should be upstreamed.

**Comment by barthalion** [comment]
Have you reached out to upstream developers about this submission??

**Comment by Steinhagen** [comment]
This works really well on my machine. Thanks!

**Comment by hfiguiere** [comment]
> Have you reached out to upstream developers about this submission??

I just filed an official issue:

https://sourceforge.net/p/guitarix/bugs/119/

File: org.guitarix.Guitarix.appdata.xml (hunk 2 above)
Comment (hfiguiere) [reply org.guitarix.Guitarix.appdata.xml:2]: Fixed.

**Comment by hfiguiere** [comment]
bot, build org.guitarix.Guitarix

**Comment by Steinhagen** [comment]
> > Have you reached out to upstream developers about this submission??
> 
> I just filed an official issue:
> 
> https://sourceforge.net/p/guitarix/bugs/119/

It's good to see that it received the blessing of the upstream developers.

That being said, I haven't encountered any issues so far with the latest build.

File: org.guitarix.Guitarix.json (hunk 1 above)
Comment (hfiguiere) [reply org.guitarix.Guitarix.json:19]: it is actually needed.

**Comment by hfiguiere** [comment]
bot, build org.guitarix.Guitarix

**Pull request closed without merging**